AUTHORS

```diff
@@ -629,6 +632,7 @@ CONTRIBUTORS:
     Trevor Bautista <trevrb@trevrb.net>
     Trey Dockendorf <treydock@gmail.com>
     Troels Nørgaard <tnn@tradeshift.com>
+    tstabrawa <tstabrawa@users.noreply.github.com>
```

Comment:
This does not look like a good address. :)

Reply:
Clear intentions :)

Reply:
We've got a couple like this, which are fine.